### PR TITLE
feat: Complete test accounts (SC-3610)

### PR DESCRIPTION
**Pull request by deluca-mike**
- remove interfaces from interfaces

# Description

# Integrations Checklist

- [ ] Have any function signatures changed? If yes, outline below.
- [ ] Have any features changed or been added? If yes, outline below.
- [ ] Have any events changed or been added? If yes, outline below.
- [ ] Has all documentation been updated?

# Changelog

### CODE REVIEW

File: contracts/DebtLocker.sol [hunk 1]
```diff
@@ -48,46 +48,46 @@ contract DebtLocker is IDebtLocker {
     function claim() external override isPool returns (uint256[7] memory) {
 
         uint256 newDefaultSuffered   = uint256(0);
-        uint256 loan_defaultSuffered = loan.defaultSuffered();
+        uint256 loan_defaultSuffered = ILoanLike(loan).defaultSuffered();
```
Comment (lucas-manuel) [inline contracts/DebtLocker.sol:51]: I think we determined before that it was less gas efficient to cast immutable addresses to interfaces inline, maybe we should compare this to caching to a local interface variable. I also think this is pretty out of scope for this PR.

File: contracts/interfaces/ILoanLike.sol [hunk 2]
```diff
@@ -15,7 +15,7 @@ interface ILoanLike is IERC20 {
 
     function interestPaid() external view returns (uint256);
 
-    function liquidityAsset() external view returns (IERC20);
+    function liquidityAsset() external view returns (address);
 
     function principalPaid() external view returns (uint256);
```
Comment (lucas-manuel) [inline contracts/interfaces/ILoanLike.sol:21]: Technically this should be set up like so:

```js
// SPDX-License-Identifier: AGPL-3.0-or-later
pragma solidity 0.6.11;

import { IERC20 } from "../../modules/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";

interface ILoanLike is IERC20 {
    function amountRecovered() external view returns (uint256);
    function defaultSuffered() external view returns (uint256);
    function excessReturned() external view returns (uint256);
    function feePaid() external view returns (uint256);
    function interestPaid() external view returns (uint256);
    function liquidityAsset() external view returns (address);
    function principalPaid() external view returns (uint256);
    function updateFundsReceived() external;
    function withdrawableFundsOf(address) external view returns (uint256);
    function withdrawFunds() external;
    function triggerDefault() external;
}
```

Comment (lucas-manuel) [reply contracts/interfaces/ILoanLike.sol:21]: Posted in discord though to get a consensus

**Review by lucas-manuel — CHANGES_REQUESTED** [review]

File: contracts/DebtLocker.sol (hunk 1 above)
Comment (deluca-mike) [reply contracts/DebtLocker.sol:51]: Casting to an interface is free. Its actually wasteful to declare a variable that is a interface-casted version of an address as a separate stack variable. Also, part of a complete test accounts is such that no function requires or returns an interface, or else we are going to have problems in integration testing.

Comment (deluca-mike) [reply contracts/DebtLocker.sol:51]: Bytecode is going to be the same. But I can test to make sure.

**Review by lucas-manuel — DISMISSED** [review]
